src/main/java/uk/gov/hmcts/reform/civil/service/InitiateGeneralApplicationService.java

```diff
@@ -74,6 +94,10 @@ public class InitiateGeneralApplicationService {
     public static final String INVALID_UNAVAILABILITY_RANGE = "Unavailability Date From cannot be after "
         + "Unavailability Date to. Please enter valid range.";
 
+    private static final List<CaseState> statesBeforeSDO = Arrays.asList(PENDING_CASE_ISSUED, CASE_ISSUED,
+            AWAITING_CASE_DETAILS_NOTIFICATION, AWAITING_RESPONDENT_ACKNOWLEDGEMENT, CASE_DISMISSED,
```

Comment:
@deepthidoppalapudihmcts validate with carrie on these states and potentially ask whether this will change later in the life cycle after going live